configs/config.toml.SAMPLE

```diff
@@ -34,3 +34,8 @@
 # This is an optional parameter, if not provided concurrency level for TSS protocol
 # will be set to `3` which is a minimum to run pre-parameters generation process.
```

Comment:
Is it still valid explanation after https://github.com/keep-network/keep-ecdsa/pull/347

Reply:
If the value is <3 it will be handled by tss-lib and set to `3` anyways.

Reply:
For 3 how many concurrent generations are started? Worth explaining here what 3 means.

Reply:
Or... we can remove it now with the fix in `tss-lib`.

Reply:
We no longer need this property at all as the bug on tss-lib side was fixed https://github.com/keep-network/keep-ecdsa/pull/347